crypto/batchverifier.c

```diff
@@ -0,0 +1,28 @@
+#include "sodium.h"
+int ed25519_batch_wrapper(const unsigned char *messages1D,
+                          const unsigned long long *mlen,
```

Comment:
does `long long` correspond to go's `int` for both 32 and 64 bit arch? It is not, `long long` is 64 and `int` is 32 or 64 depending on an arch.

Reply:
right, I missed the fact these are converted and copied before calling the wrapper.
```
	messageLens := make([]C.ulonglong, len(msgLengths))
	for i, l := range msgLengths {
		messageLens[i] = C.ulonglong(l)
	}
```